internal/statistics.go

```diff
@@ -140,38 +139,30 @@ func (s *statistics) sendStatsToOwner(stats *bytes.Buffer) {
 	}
 }
 
-func (s *statistics) registerMetrics() {
+func (s *statistics) collectMetrics() {
+	s.collectOSMetrics()
+	s.collectRuntimeMetrics()
+}
+
+func (s *statistics) collectRuntimeMetrics() {
 	var m runtime.MemStats
 	runtime.ReadMemStats(&m)
```

Comment:
does this call have an error code return possibility?

Reply:
No this function does not return an error.